crates/astria-composer/src/searcher/executor/mod.rs

```diff
@@ -165,7 +166,7 @@ impl Executor {
     }
 
     /// Create a future to submit a bundle to the sequencer.
-    #[instrument(skip(self), fields(nonce.initial = %nonce))]
+    #[instrument(skip_all, fields(nonce.initial = %nonce, rollup_counts = %ReportBundleRollupIdCounts(&bundle)))]
```

Comment:
I think we should pay the price for this in `BundleFactory` while constructing these bundles, similar to what I have done here: https://github.com/astriaorg/astria/pull/807